### ISSUE TITLE
add confetti falling animation

### ISSUE THREAD

**Issue by Prriya-Karn** [ACMW-Manipal/acmw-website-24#38]
in application form:-
1. when submit the form then show  confetti falling animation, so that attractive page will show 
2. and also background color of application form is not matched with overall website background color

**Comment by Prriya-Karn** [comment]
I would like to work on this issue please assign to me:)

**Comment by Prriya-Karn** [comment]
please assign this issue to me:)

**Comment by Prriya-Karn** [comment]
I sent PR please review it:)

**Comment by Prriya-Karn** [comment]
I sent PR please review it:)

**Comment by Prriya-Karn** [comment]
I sent PR please review it:)

**Comment by prernamittal** [comment]
Hey I checked.  Are you sure the confetti is working? I couldn't see it. Also we're constantly reviewing your work, please avoid spamming :)

**Comment by Prriya-Karn** [comment]
Yes I added the confetti functionality, may be there was some issue during commit and sorry for spamming.. my network was not good so I thought message is not sent.. sorry for that 🥲

**Comment by prernamittal** [comment]
> Yes I added the confetti functionality, may be there was some issue during commit and sorry for spamming.. my network was not good so I thought message is not sent.. sorry for that 🥲

No problem, could you please attach an ss of this feature, I still can't see it
